SDBLV/beliveo-files
# add menu to apps

#### What does this PR do?
- add a menu in the index.php, 

#### Where should the reviewer start?
- On the beliveo files repository, into components you can find the file menu-bar

#### How should this be manually tested? (if applies)
to see if this going to work, try this
1. :one: add the next line on the index.php 
include '../../node_modules/beliveo-files/components/menu-bar.php';
2. :two: copy the file on the repository app-template->node_modules->beliveo-files->components and add copy the image menu-icon.png from the repository beliveo-files, into images of about-template
3. :tada: now go to browser and type localhost/app-template, you should be able to see the icon for the menu-apps, click and this gona deploy a sub menu
